### PR TITLE
Icyscools/#20 Add Legislative Process Page: Body Skeleton

### PR DESCRIPTION
**Pull request by Icyscools**
Related to #20.

This PR intended to add legislative process heading text and their styles according to this [Figma design](https://www.figma.com/file/cdtZnPegHm5CGHWqIND5Im/Design_Phase01?type=design&node-id=319-155076&mode=dev).

- [x] Add a Blue Title section. Use the current timestamp for the updated date for now.
- [x] Add every heading 1 and heading 2 texts.
- [x] The body should be responsive.

Note that: I set `max-width` equals to 800px as same as this design's body width on desktop size, but not sure if we have any specific breakpoint.

Example
---

Viewport of 320px width.

image

---

Viewport of 800px width.

image

---

Viewport of 1024px width.

image

### CODE REVIEW

File: src/routes/legislative-process/+page.svelte [hunk 1]
```diff
@@ -1 +1,86 @@
...
+		<div class="w-full max-w-[800px] px-10 py-10 md:py-20">
+			<h1 class="fluid-heading-05 mb-4">ร่างกฎหมายกลายเป็นกฎหมายได้อย่างไร</h1>
+			<p>
+				<span class="text-text-primary font-bold">อัปเดตล่าสุด :</span>
+				<span class="text-helper-text-01">{dayjs(new Date()).format('DD MMM BBBB')}</span>
+			</p>
+		</div>
+	</header>
+	<div class="w-full max-w-[800px] flex flex-col gap-10 px-4 md:px-10 py-8 bg-ui-01 md:bg-white">
+		<section class="flex flex-col gap-4 bg-white px-4 md:px-0 py-8 md:py-0">
```
Comment (Th1nkK1D) [inline src/routes/legislative-process/+page.svelte:21]: You might notice that we use the same class for every section, h1, div, 52, hr. To clean this up, I think we have 2 choices.

1. We should be able to use Tailwind's `@apply` together with Svelte's `style` tag to add the same class to every element as follows:

```svelte
<style>
section {
  @apply flex flex-col gap-4 bg-white px-4 md:px-0 py-8 md:py-0;
}
/* other elements */
</style>
```
2. Split those duplicated codes into a new component (can be created in `/components/legislative-process`), which receives props or a slot to render the content.

**Review by Th1nkK1D — CHANGES_REQUESTED** [review]
Nice! One suggestion for the layout. Even though the content is aligned left, the container (with max-width) should still be in the middle. It will feel more natural. Right now it's on the left. We want space on both the left and right to be the same when the window is bigger than max-width.

image

File: src/routes/legislative-process/+page.svelte (hunk 1 above)
Comment (Icyscools) [reply src/routes/legislative-process/+page.svelte:21]: Agreed! I might working on the first choice to clean up.

**Comment by Icyscools** [comment]
Done!

Now the content body should be in the center if the display screen has a width of more than 800px. And, I cleaned up the styles as you mentioned. But somehow I can't `@apply` our custom classes (`fluid-heading-04`, `fluid-heading-05`) so I remain custom classes in `class` property of tag.

Example viewport of 1024px width.

image

**Comment by Th1nkK1D** [comment]
> Done!
> 
> Now the content body should be in the center if the display screen has a width of more than 800px. And, I cleaned up the styles as you mentioned. But somehow I can't `@apply` our custom classes (`fluid-heading-04`, `fluid-heading-05`) so I remain custom classes in `class` property of tag.

LGTM! You're right. Since both of those classes are not from Tailwind, we can't use `@apply`. Thank you for another PR ;)

**Review by Th1nkK1D — APPROVED** [review]

**Pull request merged**